### PR TITLE
Update questions from the mobility appraisal form

### PR DESCRIPTION
**Pull request by theharq**
Update the appraisal form for `Mobility` ( Promoting Opportunity Award)

### CODE REVIEW

**Comment by theharq** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;IuBio3Xp&#x2F;1205-qae16imp-promoting-opportunity-edit-appraisal-questions"> [QAE16IMP] Promoting Opportunity - edit appraisal questions</a>

File: lib/tasks/temprary/update_mobility_forms.rake [hunk 1]
```diff
@@ -0,0 +1,27 @@
+namespace :db do
+  desc "remove keys from appraisal forms no longer used"
+  task remove_keys_from_mobility_assessor_assignments: :environment do
+
+    mobility_appraisals = AssessorAssignment.joins(:form_answer)
+      .where(award_year: AwardYear.find_by_year(2017),
+             position: [0, 1, 4],
+             form_answers: {award_type: "mobility"})
```
Comment (rusllonrails) [inline lib/tasks/temprary/update_mobility_forms.rake]: @theharq probably you need only already assessed assessments, not all:
```
where("assessed_at IS NOT NULL")
```

**Comment by rusllonrails** [comment]
@theharq looks fine, small comment about rake script and we are waiting for confirmation from client side

**Comment by rusllonrails** [comment]
@theharq all fine, going to merge

**Pull request merged**